rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -1098,7 +1104,7 @@ def _load_model(
             data_signature=model_data_example.get_signature(),
             label_data=label_data,
             entity_tag_specs=entity_tag_specs,
-            config=meta,
+            config=copy.deepcopy(meta),
```

Comment:
this fix was needed as this config gets passed to tensorflow which in place convert all values of the dictionary to tensorflow types (e.g. it will replace an ordinary python string with a tensorflow specific string). to avoid tf modifying this original dictionary we need to pass in a copy...